yt/visualization/plot_modifications.py

```diff
@@ -405,15 +423,13 @@ class QuiverCallback(PlotCallback):
     substantial variation in field strength.
     """
     _type_name = "quiver"
-    _supported_geometries = ("cartesian", "spectral_cube", "cylindrical-2d")
+    _supported_geometries = ("cartesian", "spectral_cube", "polar", "cylindrical")
     def __init__(self, field_x, field_y, factor=16, scale=None,
-                 scale_units=None, normalize=False, bv_x=0, bv_y=0,
```

Comment:
Why did you remove these two keyword arguments? Since they're exposed in the user-facing API I think we need to keep them.

Reply:
This is because `plot.data.ds.coordinates.pixelize` does not accept `plot.data[self.field_x]`, but only `self.field_x`, so I can not subtract `bv_x` from the field `self.field._x`. Or you have any idea?

Reply:
You could add those as optional new arguments to `pixelize`?

Reply:
Sure! I'll take a look at it.